### DISCUSSION TITLE
Possible to define conditions to display a template based on the "Template" selection for a page (not post)

### DISCUSSION THREAD

**Discussion by espechtcfe** [elementor/elementor#27325] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

We're migrating an older site to use Hello Elementor as the parent theme while preserving some metadata that has already been collected on the site.  When we edit a Page in Wordpress (not in Elementor), there is a spot where we can select "Template" for the page (see screenshot provided).  In our current theme, in PHP we use the selection of Template to determine which user-facing template to use for the page.
In the Theme Builder, when you are defining conditions to display a template for a page (not a post), there is currently not an option to
display a template based on meta such as the "Template" that I provided in the screen shot. Is there a way to choose the template based on that metadata?   I asked Elementor support and they said that this isn't supported functionality, but that I might be able to ask here for a solution.
![page_attribute_template](https://github.com/elementor/elementor/assets/42548009/123c73f1-c84d-4bb5-bfa7-1713b59bf985)

### Describe the solution you'd like.

We're hoping there may be a workaround that allows us to choose the Elementor template based on the selection of Template for the Page in Wordpress.

### Describe alternatives you've considered.

If there isn't a way to do it with Elementor, we might have to migrate our existing data to a taxonomy.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by majojendruch** [comment]
Hello @espechtcfe I am facing similar problem. Have you been able to find a workaround, until this feature will be (hopefully) implemented to Elementor? +1 for your request!
